packages/edit-site/src/components/sidebar-navigation-screen-navigation-menus/style.scss

```diff
@@ -61,3 +61,7 @@
 		}
 	}
 }
+
+.edit-site-sidebar-navigation-screen-navigation-menus-off-canvas-container.is-single-page-list .block-editor-list-view-block-select-button__lock {
+	display: none;
```

Comment:
Do we want to remove the locks only when it's "single page list". In other words, I'm wondering whether `is-single-page-list` classname is necessary.

Reply:
If it should be a global change, maybe we can also avoid the extra div. As you can see, a few lines above in this style file, we override other off canvas menu styles as well.

Reply:
Meanwhile, it was applied globally, and this rule was removed.